lib/rubocop/cop/salsify/style_dig.rb

```diff
@@ -1,5 +1,3 @@
-# encoding: utf-8
-
 module RuboCop
   module Cop
     module Salsify
```

Comment:
While we're in here can we add a reference to https://github.com/bbatsov/rubocop/issues/5332?